tests/controllers/test_category.py

```diff
@@ -0,0 +1,189 @@
+import pytest
+
+from main.engines.categories import add_category
+from main.engines.users import add_user
+from main.models.category import CategoryModel
+from main.models.user import UserModel
+
+mock_email = "email@gmail.com"
+mock_password = "random_string"
+
+
+def prepare_user(email: str = mock_email, password: str = mock_password):
+    return add_user(email=email, password=password)
+
+
+def prepare_category(user: UserModel):
+    return add_category(name="Cat 1", creator_id=user.id)
+
+
+async def get_access_token(client, user: UserModel):
+    return (
+        (
+            await client.post(
+                "/login",
+                json={"email": user.email, "password": mock_password},
+            )
+        ).json()
+    )["access_token"]
```

Comment:
We can simply call `create_access_token_from_id` directly

Reply:
Fixed